public/init_vic.json

```diff
@@ -11,7 +11,7 @@
             "name": "Victorian Government",
             "type": "ckan",
             "url": "http://www.data.vic.gov.au/data",
-            "filterQuery": "res_format:wms",
+            "filterQuery": ["fq=res_format%3awms"],
```

Comment:
I think it would be a lot cleaner to introduce a separate `query` property to `CkanGroupViewModel`, in addition to `filterQuery`, and handle the URL encoding automatically.  Both properties could still optionally be arrays if that's helpful.